src/windows/Button.tsx

```diff
@@ -45,6 +47,11 @@ export class Button extends ButtonBase implements React.ChildContextProvider<But
 
     private _isFocusedWithKeyboard = false;
 
+    //Offset to show context menu using keyboard.
+    protected _contextMenuOffset() {
```

Comment:
Method names should all begin with a verb. This should be _getContextMenuOffset().

Reply:
Fixed naming.